test/core/reporters/helpers/incomplete-fallback-msg.js

```diff
@@ -0,0 +1,21 @@
+
+describe('helpers.incompleteFallbackMessage', function() {
+	'use strict';
+	before(function() {
+		axe._load({
+			messages: {},
+			rules: [],
+			data: {
+				incompleteFallbackMessage: function anonymous() {
+					return 'Dogs are the best';
+				}
+			}
+		});
+	});
+
+	it('should return a string', function() {
```

Comment:
This test is sorta useless since we're not injecting data or transforming the string, but still nice to have.